backend/src/data/repositories/user-details/user-details.repository.ts

```diff
@@ -61,6 +62,19 @@ class UserDetails {
       .execute();
   }
 
+  public updateMoneyBalance(
+    userId: number,
+    newMoneyBalance: number,
+  ): Promise<UserDetailsWithMoneyBalanceDto | null> {
+    return this.#UserDetailsModel
+      .query()
+      .patchAndFetchById(userId, {
```

Comment:
Is it just a misleading in arg naming or you're trying to update `user_details` record by `id` col with `user_id`?

Reply:
Oh, realy bad name, thanks!